docs/AboutUs.adoc

```diff
@@ -10,46 +10,29 @@ We are a team based in the http://www.comp.nus.edu.sg[School of Computing, Natio
 
 == Project Team
 
-=== John Doe
-image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== David Ten
+image::davidTen.png[width="150", align="left"]
+{empty}[https://github.com/davidten[github]]
 
-Role: Project Advisor
+Role: Team Lead + Scheduling and Tracking + CRM Expert
+Responsibilities: LinkedIn Integration
```

Comment:
Remember to use asciidoctor compatible code to do line breaks -- current version is in a single line